docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -15,6 +15,8 @@
 - dubbo
 - dubbo-2.7.x
 - dubbo-3.x
+- dubbo-threadpool
+- dubbo-threadpool-2.7.x
```

Comment:
I think these two are not necessary.

Reply:
Instead, `Supported-list.md` should be updated.

Reply:
I set different keys in `skywalking-plugin`, so the instruments could be excluded optionally. I think the `Plugin-list.md` is related to those keys, we don't need to add these?

```properties
dubbo-2.7.x=org.apache.skywalking.apm.plugin.asf.dubbo.DubboInstrumentation
dubbo-threadpool-2.7.x=org.apache.skywalking.apm.plugin.asf.dubbo.AbstractServerInstrumentation
```

Reply:
I set different keys in `skywalking-plugin`, so the instruments could be excluded optionally. I think the `Plugin-list.md` is related to those keys, we don't need to add these?

```properties
dubbo-2.7.x=org.apache.skywalking.apm.plugin.asf.dubbo.DubboInstrumentation
dubbo-threadpool-2.7.x=org.apache.skywalking.apm.plugin.asf.dubbo.AbstractServerInstrumentation
```

Reply:
OK. Could you explain the reason of this new key? Are you going to provide user a way to disable part of the plugin?

Reply:
Yes, now skywalking mix trace and metrics instrumentation together. I think it would better if there has choice for users. 

This case make them together or separated are okay, like connect-pool druid should be separate, because most time we don't need the trace connection operation but need the metrics. Maybe I will create another PR to do this.

Reply:
Make sense. If we are going this way, we should make a little change to `Plugin-list.md` file.

Separate the plugins of tracing and metrics to make users clear. And there is `check-javaagent-plugin-list.sh` that needs to be updated to adopt the change as well.

Reply:
> there is `check-javaagent-plugin-list.sh` that needs to be updated to adopt the change as well.

This script doesn't need to be updated. It works well.